share/doc/wake/basic_wake_tutorial.md

```diff
@@ -0,0 +1,858 @@
+## Wake Tutorial
+
+This tutorial will teach you the basics of Wake language. After completeing this tutorial, you will have typed and executed many basic wake functions.
+
+### Invoking wake
+
+1. Create a folder inside `/scratch` directory. `mkdir /scratch/<your_name>/wake_tutorial`.
```

Comment:
wake is a public open source project

/scratch is not a typical directory that exists on a machine

Try suggesting another directory, like in the users `$HOME` or in `/tmp`

Reply:
Okay